internal/beater/middleware/timeout_middleware_test.go

```diff
@@ -66,6 +66,7 @@ func TestTimeoutMiddleware(t *testing.T) {
 			r := c.Request.WithContext(ctx)
 			c.Request = r
 			time.Sleep(2 * timeout) // let the context expire
```

Comment:
```suggestion
			time.Sleep(2 * timeout) // let the context expire
```
do we still need this line?